nouns.json

```diff
@@ -40,6 +44,7 @@
         "RU": { "atom": "нётерова"    , "pstn": "before" },
       },
       "433b071e": {
+        "CA": { "atom": "localmentnoetherià"    , "pstn": "after"  },
```

Comment:
should there be a space in between "localment" and "noetherià"?

Reply:
Yes, it must be "localment noetherià". Sorry!